scope.go

```diff
@@ -49,9 +53,12 @@ type scope struct {
 	labels prometheus.Labels
 }
 
-func newScope(req *http.Request, u *user, c *cluster, cu *clusterUser) *scope {
-	h := c.getHost()
+func newScope(req *http.Request, u *user, c *cluster, cu *clusterUser, sessionId string) *scope {
 
+	h := c.getHost()
```

Comment:
should we use if-else here to avoid unnecessary counter's increase on line 58 when session_id isn't empty?